src/unicode.cpp

```diff
@@ -8,6 +8,11 @@ ADA_PUSH_DISABLE_ALL_WARNINGS
 ADA_POP_DISABLE_WARNINGS
 
 #include <algorithm>
+#if ADA_NEON
+#include <arm_neon.h>
+#elif ADA_SSE2
+#include <emmintrin.h>
+#endif
```

Comment:
Can we have a `simd.h` header file which contains 11-15 lines? We might end up adding these checks to lots of places?

Reply:
I recommend waiting for the time when do use SIMD in a lot of places to do the refactoring.